### PR TITLE
UI Text for mobile ESLint plugin GraphQL warnings

### PR DESCRIPTION
**Pull request by clivewong**

### CODE REVIEW

File: src/rules/apex/apex-import.ts [hunk 1]
```diff
@@ -24,11 +24,11 @@ export const rule = ESLintUtils.RuleCreator.withoutDocs({
     meta: {
         docs: {
             description:
-                'Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios. See https://developer.salesforce.com/docs/atlas.en-us.mobile_offline.meta/mobile_offline/apex.htm for more details.'
+                'Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios. See Use Apex While Mobile and Offline (https://developer.salesforce.com/docs/atlas.en-us.mobile_offline.meta/mobile_offline/apex.htm) for more details.'
         },
         messages: {
             [APEX_IMPORT_RULE_ID]:
-                'Using Apex in LWC Offline-enabled mobile apps requires careful consideration.'
+                'Using Apex in LWC Offline-enabled mobile apps requires careful consideration in offliine scenarios.'
```
Comment (ben-zhang-at-salesforce) [inline src/rules/apex/apex-import.ts:31]: offliine scenarios -- > offline scenarios ?

Comment (khawkins) [reply src/rules/apex/apex-import.ts:31]: I'll make an easy-to-incorporate "suggestion" for this one. 🙂

```suggestion
                'Using Apex in LWC Offline-enabled mobile apps requires careful consideration in offline scenarios.'
```

**Review by khawkins — APPROVED** [review]
Looks good to me overall!

File: src/rules/apex/apex-import.ts (hunk 1 above)
Comment (clivewong) [reply src/rules/apex/apex-import.ts:31]: @ben-zhang-at-salesforce I wanted to provide more context to the user. But if adding "in offline scenarios." to the message isn't helpful, I don't mind removing it.

Comment (clivewong) [reply src/rules/apex/apex-import.ts:31]: @ben-zhang-at-salesforce D'oh! Thanks for catching that!

**Pull request merged**